frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -799,6 +818,7 @@ class TableInteractive extends Component {
         }}
       >
         <HeaderCell
+          c={cellColor}
```

Comment:
This doesn't seem to have any effect because the color is set by a more specific component https://github.com/metabase/metabase/blob/aaf8766a87ea3dff643363e22c510d87f7b24c7e/frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx#L122

Does that mean we could remove `cellColor` here? Also, I'm not totally sure where `text-medium` is from.

Reply:
@WiNloSt The cellColor is needed for the `isSorted` case, when the cell's column is sorted.

```
.TableInteractiveHeaderCellDataSorted {
  color: var(--mb-color-brand);
}
```

For `text-medium` it is from HeaderCell:

export const HeaderCell = styled.div`
  color: ${color("text-medium")};

  &:hover {
    color: ${color("text-brand")};
  }
`;

Reply:
I just noticed that the cell header color is always the brand color, so I think this is also a no-op in the existing codebase 😅; yes we can get rid of the cellColor in HeaderCell.

![CleanShot 2567-05-15 at 00 14 16@2x](https://github.com/metabase/metabase/assets/4714175/1a7e0fab-3d93-4cf7-b706-f95d932b8bc8)